.github/workflows/coding-standards.yml

```
@@ -86,10 +85,12 @@ jobs:
           path: .cache/phpcs.json
           key: ${{ runner.os }}-date-${{ steps.get-date.outputs.date }}-phpcs-cache-${{ hashFiles('**/composer.json', 'phpcs.xml.dist') }}
 
+      # Since Composer dependencies are installed using `composer update` and no lock file is in version control,
+      # passing a custom cache suffix ensures that the cache is flushed at least once per week.
       - name: Install Composer dependencies
         uses: ramsey/composer-install@83af392bf5f031813d25e6fe4cd626cdba9a2df6 # v2.2.0
         with:
-          composer-options: "--no-progress --no-ansi"
+          custom-cache-suffix: $(/bin/date -u --date='last Mon' "+%F")
```

Considering the date is still retrieved in a separate step anyway for the PHPCS cache, why not re-use it ?

```suggestion
          custom-cache-suffix: ${{ steps.get-date.outputs.date }}
```

As a side-note: what with the two PHPCS runs in this workflow and these not using separate PHPCS caches, the caching of the PHPCS cache file is kind of useles....

---

I did some testing and these were my findings:

Scenario 1: Run `phpcs -n` first
- first run: 28.41s
- second run: 1.56s
- first run of `phpcs tests`: 678ms
- second run of `phpcs tests`: 533ms
- rerunning `phpcs -n`: 879ms

Scenario 2: Run `phpcs tests` first
- first run: 16.16s
- second run: 331ms
- first run of `phpcs -n`: 13.4s
- second run of `phpcs -n`: 1.43s
- rerunning `phpcs tests`: 403ms

I'm not familiar with how PHPCS generates the cache file, but it seems that there's still a benefit to caching `.cache/phpcs.json`, even if the cache file does not include _everything_ we would like to be cached.

Let's open a different ticket to discuss further if we want to perform a deeper dive.

---

Would you mind opening the issues for it ? I can send in a PR to fix it. We'd only need to use different cache file names to get the optimal benefit in CI and that can be sorted via CLI arguments.

---

Done! https://core.trac.wordpress.org/ticket/57148